### ISSUE TITLE
audio input not switching

### ISSUE THREAD

**Issue by dombarnes** [ttscoff/bunch#289]
I'm attempting to use Bunch for a simple cough button. My bunch script is as follows:
```
---
title: Cough Button
shortcut: m
---
(audio input mute)
(notify Muted)
!(audio input macbook)
!(notify Unmuted)
```
The mute command works successfully when I toggle on. When I toggle off, the notification fires, but I am not unmuted. I can resolve by changing 
Audio inputs are as follows:
![image](https://user-images.githubusercontent.com/511773/217387789-2cba6721-444d-40c7-a3a3-46e23e8e4b02.png)

This works fine if i just set the input audio level
```
(audio input volume 0)
!(audio input volume 50)
```

**Comment by dombarnes** [comment]
Was doing some more testing and `!(audio input unmute)` seems to do the right thing. Not sure if this is documented anywhere.

**Comment by ttscoff** [comment]
Yeah, you have to use `unmute` or set a non-zero volume level, otherwise 
you're just setting the input device. Apparently I didn't mention unmute 
in docs, will fix.

-Brett

**Comment by dombarnes** [comment]
Thanks for clarifying! I also found that setting input volume to 0 didn’t seem to mute the audio either. Maybe that’s just an oddity in macos x.On 8 Feb 2023, at 13:47, Brett Terpstra ***@***.***> wrote:﻿
Yeah, you have to use `unmute` or set a non-zero volume level, otherwise 
you're just setting the input device. Apparently I didn't mention unmute 
in docs, will fix.

-Brett
